README.md

```diff
@@ -9,6 +9,14 @@ A Config Controller
 Using YAML files *endpoints* can be defined to be exposed or called on specified conditions. The call of an endpoint results in *coco* calling the same endpoint on a given set of hosts. *coco* can be configured to check or compare the results and based on that export [prometheus](prometheus.io) alerts, write [slack](slack.com) messages or call other endpoints.  
 The values passed to *endpoints* can be marked as part of a global state, which is tracked and kept in sync on the hosts.
 
+## Metrics
+Prometheus metrics are exported at a port specified in the config (`metrics_port`).
+These are:
+- `coco_requests` (labels: `['endpoint']`): Total number of requests received per endpoint.
```

Comment:
both metrics end in `_total` (that's what `prometheus_client` if you set a `unit`)